.circleci/config.yml

```diff
@@ -15,9 +15,22 @@ jobs:
           command: git submodule update --init
       - run: apt update
       - run: apt install -y zip
+      - run: make prepare
```

Comment:
Is this needed? Shouldn't make release and make call it anyway?

Reply:
no, it's not needed but it's a nice test since we have the target now